godaddy/coblocks
# Examples: Update example images with those from WP core.

#1106 introduced WikiMedia images with non-GPL-compatible licenses. This PR swaps them out for WordPress core's own Public Domain example images, which are all hosted on the WP.org CDN.

In https://github.com/WordPress/gutenberg/pull/17493, @jasmussen also points out https://commons.wikimedia.org/wiki/Featured_Pictures_in_the_Public_Domain, giving CoBlocks other options to this PR too.

These images are all licensed under one form or another of the CC Attribution-based licenses, meaning  "You must give appropriate credit, provide a link to the license, and indicate if changes were made."
  - https://commons.wikimedia.org/wiki/File:St_Pancras_Railway_Station_2012-06-23.jpg
  - https://commons.wikimedia.org/wiki/File:Headey,_Lena_(2007).jpg
  - https://commons.wikimedia.org/wiki/File:Beautiful_river_landscape_in_the_fall.jpg (PD)
  - https://commons.wikimedia.org/wiki/File:Landscape_of_Jackson_Hole,_October_2010.jpg
  - https://commons.wikimedia.org/wiki/File:California_Drought_Dry_Riverbed_2009.jpg (PD)
  - https://commons.wikimedia.org/wiki/File:Taiga_Landscape_in_Canada.jpg
  - https://commons.wikimedia.org/wiki/File:Lesotho_Landscape.jpg
  - https://commons.wikimedia.org/wiki/File:Landscape_dunes.JPG
  - https://commons.wikimedia.org/wiki/File:WordPress_logo.svg
  - https://commons.wikimedia.org/wiki/File:Enjoying_summer.jpg

These images were actually fine, being Public Domain:
  - https://commons.wikimedia.org/wiki/File:Beautiful_river_landscape_in_the_fall.jpg
  - https://commons.wikimedia.org/wiki/File:California_Drought_Dry_Riverbed_2009.jpg

**Testing Instructions**
- Apply this branch to a local testing install.
- Open a new post.
- Using the top toolbar inserter, enter the CoBlocks subsections and hover over the galleries, Media Card, and Author blocks.
- Verify the example images all appear fine.
